source/includes/_invoice.md

```diff
@@ -4562,42 +2780,10 @@ invoiceApi.get_invoice_translation(locale, api_key, api_secret)
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-< Content-Type: text/plain
-
-sports-monthly = Voiture Sport
-```
-```java 
-sports-monthly = Voiture Sport
-```
-```ruby
-invoiceTitle=FACTURE
-invoiceDate=Date:
-invoiceNumber=Facture #
-invoiceAmount=XXXMontant à payer
-invoiceAmountPaid=XXXMontant payé
-invoiceBalance=Nouveau montant
-
-accountOwnerName=Chauffeur
-
-companyName=Killbill, Inc.
-companyAddress=P.O. Box 1234
-companyCityProvincePostalCode=Springfield
-companyCountry=USA
-companyUrl=http://killbilling.org
-
-invoiceItemBundleName=Armes
-invoiceItemDescription=Description
-invoiceItemServicePeriod=Periode de facturation
-invoiceItemAmount=Montant
-
-processedPaymentCurrency=(*) Le payment à été payé en
-processedPaymentRate=Le taux de conversion est
-```
-```python
-sports-monthly = Voiture Sport
+```json
+{
+    "sports-monthly" : "Voiture Sport"
```

Comment:
I believe a location header is returned here. Could you please check and if so remove the json response?

Reply:
Hi @reshmabidikar , checked the response section, its updated with relevant response code at all the places. 

For the above translation one, a bit unclear, there is a translation table supposed to be returned.

![image](https://github.com/killbill/slate/assets/75751774/5db22d0b-9eca-4385-a78f-833c41cf6301)


![image](https://github.com/killbill/slate/assets/75751774/53dbc04c-d205-499c-a0db-64421b91f7e1)

Reply:
I think the translation table refers to the text `sports-monthly = Voitue Sport`. So, a plain text response is returned.